client/executor/wasmtime/src/runtime.rs

```diff
@@ -253,6 +253,18 @@ fn common_config(semantics: &Semantics) -> std::result::Result<wasmtime::Config,
 	config.cranelift_opt_level(wasmtime::OptLevel::SpeedAndSize);
 	config.cranelift_nan_canonicalization(semantics.canonicalize_nans);
 
+	let profiler = match std::env::var_os("WASMTIME_PROFILING_STRATEGY") {
+		Some(os_string) if os_string == "jitdump" => wasmtime::ProfilingStrategy::JitDump,
+		Some(_) => {
+			log::warn!("WASMTIME_PROFILING_STRATEGY is set to unknown value, ignored.");
```

Comment:
As I said, this should probably be a single message. The reason I am emphasizing this is should this actually be hit, then the user will be drowned in `warn` messages.

Reply:
Ok, to make sure the warning is logged only once I used an AtomicBool, if you prefer another method, please specify :)

Reply:
It is fine, although I would put the static declaration right next to the place where we print

Reply:
> I would put the static declaration right next to the place where we print

This is the case, the static is declared just above the function that uses it.

Reply:
No, I mean EVEN closer : ) i.e. next to the `if` in the function body.

Reply:
Thanks for the explanation, it's done.